journal/202012.md

```diff
@@ -0,0 +1,320 @@
+# {DRAFT} Decred Journal – December 2020
+
+![{alt text}](../img/{file.jpg} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+{ introduction, major news of the month ordered by impact, don't put any links here }
+
+## Development
+
+Unless otherwise noted, the work reported here has the "merged to master" status. It means that the work is completed, reviewed, and integrated into the source code that advanced users can build and run, but is not yet available in release binaries for regular users.
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+Merged in master and backported to v1.6 release:
+
+- corrected [fee calculations](https://github.com/decred/dcrd/pull/2530) to fix some transactions being considered lower priority in some reorg scenarios
+- whitelisted DCP-5 [violations](https://github.com/decred/dcrd/pull/2533). Due to a bug that has since been fixed, 5 blocks were accepted with an old block version 6 when the majority of the network had already upgraded to version 7. The blocks were otherwise valid and are now whitelisted to allow fully syncing the chain without checkpoints. { is this wording good enough? is no-checkpoint sync the primary reason for whitelisting? how "bad" was the bug and which PR fixed it? }
```

Comment:
> is this wording good enough?

Yes

> is no-checkpoint sync the primary reason for whitelisting

Yes

> how "bad" was the bug and which PR fixed it?

Zero effect.  The commit is linked in the PR ([this one](https://github.com/decred/dcrd/commit/f1e004695ec5d40dc05ee62e741528248ededce6)), but I don't think it needs to be list again since it's already in the PR as mentioned.

Reply:
@davecgh thanks!

I got puzzled first since that [linked](https://github.com/decred/dcrd/commit/f1e004695ec5d40dc05ee62e741528248ededce6) commit didn't seem to change behavior with current block versions, but rather simplify and prevent future omissions. Out of curiosity I followed validate.go and I think the fix was in a [different](https://github.com/decred/dcrd/commit/d6c152c04cce5247ad1e392d09e38a848a61305f) commit that actually added code to reject version 6 blocks.

Reply:
Technically, yes, but they both happened at the exact same time, and the latter also covers it too, so we figured it was better to link the one that not only addresses it, but helps prevent future omissions.